### DISCUSSION TITLE
Is it possible to do inline rendering?

### DISCUSSION THREAD

**Discussion by glyh** [typst/typst#4274] [Q&A] [answered]
I would like to have typst built into some note taking apps. But I need "inline rendering", like what inline formula would look like in markdown. 

For example, if I type out code corresponding to a grid, I would like the rendered result to only include a grid with no padding, headers, footers or whatsoever.

Is it possible to achieve with current typst? 

Thank you!

**Comment by LaurenzV** [comment, marked as answer]
Does doing `#set page(margin: 0pt, height: auto, width: auto)` at the beginning of the page achieve what you want?

**Reply by glyh** [reply]
Not really if you test it out: 

![image](https://github.com/typst/typst/assets/12870742/80ca9e5c-49d8-49e7-96fd-e8aa6a64ea1d)

Some output is sliced. Looks like it slices at where the text should sit on. I guess this is a bug? Or is this intended?

**Reply by Enivex** [reply]
> Some output is sliced. Looks like it slices at where the text should sit on. I guess this is a bug? Or is this intended?

Depends how you look at it I guess. Inline math has (on purpose) a smaller bounding box to reduce variation in line spacing. But it clearly has unintentional consequences in certain situations.

If you want to render something standalone like this, you should use block math instead.

**Reply by glyh** [reply]
Looks like using block math work for me, thanks!
